nextflow_schema.json

```diff
@@ -1,7 +1,7 @@
 {
     "$schema": "http://json-schema.org/draft-07/schema",
-    "$id": "https://raw.githubusercontent.com/nf-core/staramr/master/nextflow_schema.json",
-    "title": "nf-core/staramr pipeline parameters",
+    "$id": "https://raw.githubusercontent.com/phac-nml/staramrnf/master/nextflow_schema.json",
```

Comment:
Can you change the branch `master` to `main` here since that is the main branch name we use.

Reply:
b5d6426

Reply:
Missed a few [4fe0406](https://github.com/phac-nml/staramrnf/pull/20/commits/4fe0406ae17756a5e925664194e4fa256c12fe4a)